### PR TITLE
add more css to override default notebook css

### PR DESCRIPTION
**Pull request by seethroughdev**
This PR does two things.  

1.  it forces the css of tables into `table-layout: inherit` to override Jupyter's generic table css
2. adds postcss to our build process which uses cssnano.  hopefully our css will start being the same in builds if no css has changed.

However, I can't verify this change directly because I don't know how to regenerate the notebook file.  But I have verified the change in rendered html.  So hopefully it works as expected.

The only actual line of css is: https://github.com/valayDave/metaflow/compare/multiple-card-decorators-custom-components-actual-components...valayDave:bug/css-table?expand=1#diff-204b2249d81ba15aa93c195d767f281fb688853d9f3e27d05630840eac9c0d56R39

And you can validate the postcss change in the rollup config.

I also noticed there are other prettier changes in `app.css`.  I don't know if this file was saved before prettier, or we have a different css config.  we should check this later too.

### CODE REVIEW

**Review by obgibson — COMMENTED** [review]
The very first time I load a page after `yarn dev`, the DAG is brocken. I couldn't replicate this with `master`.  It fixes itself after a resize. Can you replicate?

Screen Shot 2022-01-22 at 8 49 06 AM

File: metaflow/plugins/cards/ui/src/app.css [hunk 1]
```diff
@@ -41,6 +36,7 @@ td) {
 .mf-card table {
   position: relative;
   min-width: 100%;
+  table-layout: inherit !important;
```
Comment (seethroughdev) [inline metaflow/plugins/cards/ui/src/app.css:39]: this is the only active change in this PR.

**Comment by seethroughdev** [comment]
I was unable to reproduce @obgibson issue, however, out of caution, I have decided to remove the postcss from the rollup config.  (by commenting it out).

We will turn it back on in a separate PR after release.  That way we can confirm whatever issue Brendan was seeing earlier.

The only remaining change right now (besides prettier running on the css files), is forcing table-layout on mf-card.

**Review by obgibson — APPROVED** [review]
Works for me now

**Pull request merged**